src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
 public class Duke {
+
+    private static final String LINE = "____________________________________________________________";
+    private static final String BLANK = "    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcome();
+        Scanner in = new Scanner(System.in);
+        String inputString;
+        Task[] tasks = new Task[100];
+        int numberOfTasks = 0;
+
+        while (true) {
+            inputString = in.nextLine();
+            String[] command = inputString.split(" ", 2);
+            //list command
+            if (command[0].equals("list")) {
+                doList(tasks, numberOfTasks);
+            }
+            //bye command
+            else if (command[0].equals("bye")) {
```

Comment:
it might be useful to create an enum containing the different commands and using BYE instead of "bye"